src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -94,11 +94,11 @@
  * <p> Platform threads are designated <i>daemon</i> or <i>non-daemon</i> threads.
  * When the Java virtual machine starts up, there is usually one non-daemon
  * thread (the thread that typically calls the application's {@code main} method).
- * The Java virtual machine terminates when all started non-daemon threads have
- * terminated. Unstarted non-daemon threads do not prevent the Java virtual machine
- * from terminating. The Java virtual machine can also be terminated by invoking
- * the {@linkplain Runtime#exit(int)} method, in which case it will terminate even
- * if there are non-daemon threads still running.
+ * The <a href="Runtime.html#shutdown">shutdown sequence</a> begins when all started
+ * non-daemon threads have terminated. Unstarted non-daemon threads do not prevent
+ * the shutdown sequence from commencing. Invoking the {@linkplain Runtime#exit(int)}
+ * method will start the shutdown sequence regardless of whether non-daemon threads
+ * are still running.
```

Comment:
Future down, in the "Virtual threads" section we have "Virtual threads are daemon threads and so do not prevent the Java virtual machine from terminating". That will need to be updated too to say that it they do not prevent the shutdown sequence from commencing.

Reply:
OK I've updated this too.